pkgs/by-name/te/testkube/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "testkube";
+  version = "2.1.56";
+
+  src = fetchFromGitHub {
+    owner = "kubeshop";
+    repo = "testkube";
+    rev = "v${version}";
+    sha256 = "sha256-P+A9lUMzQ3M0SEVZBMDSMj8S0uCsEhadv5vDRxbQORA=";
+  };
+
+  vendorHash = "sha256-44aIwddMH6CMfTno90xGkHgna4DO2Ii3KhpMwv6Zjmo=";
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.builtBy=nixpkgs"
+    "-X main.commit=v${version}"
+    "-X main.date=1970-01-01-00:00:01"
+  ];
+
+  doCheck = false;
+  subPackages = [ "cmd/kubectl-testkube" ];
+
+  meta = with lib; {
+    description = "Kubernetes-native framework for test definition and execution";
+    homepage = "https://github.com/kubeshop/testkube/";
+    license = licenses.mit;
+    mainProgram = "kubectl-testkube";
+    maintainers = with maintainers; [ ];
```

Comment:
new packages must have maintainers.

Reply:
fixed in 8b389a095b1cc04db34d2f6f0e5e524270ba7e04